mod/thewire/start.php

```diff
@@ -326,7 +326,12 @@ function thewire_setup_entity_menu_items(\Elgg\Hook $hook) {
 	}
 	
 	$menu = $hook->getValue();
-	$menu->remove('edit');
+	
+	$enable_editing = (bool) elgg_get_plugin_setting('enable_editing', 'thewire');
+	
+	if(!$enable_editing) {
```

Comment:
Same suggestions as above, you don't re-use `$enable_editing` so check the plugin setting in the `if`